_posts/2016-03-07-shaping-an-open-source-editorial-team.md

```diff
@@ -0,0 +1,72 @@
+
+Hoodie is a group of fun folks who created a bit of code and decided to make it available to anyone with an internet connection and a sense of adventure. Instead of keeping their code to themselves, and limiting its capabilities to just a certain set of brains, they opened it up to the entire internet (and world) so that endless contributors can help shape and grow all the different ways Hoodie can now be useful.
+
+That approach, instead of a more proprietary one, is what people in the tech industry call “Open Source.”
+
+The folks at Hoodie are a little different though, did I mention adventure? They know that there’s a whole world outside of this tech bubble, where folks with millions of different talents help make amazing things every day. So why would they want to limit their open source community to just the kind of contributor that knows how to write computer code?
+
+In comes the __Hoodie editorial team__. The editorial team began as a [discussion in October 2015](https://github.com/hoodiehq/discussion/issues/82). The original thought was to first, recruit members of the team, who could then be responsible for planning and writing blog posts on or around Hoodie, Hoodie releases, contributors, side projects, etc. There was also some mention of coordinating Twitter and helping with support, but we’ll get to that.
+
+After much social media-ing and direct messaging, the editorial team officially started on the 21st of November, 2015 with an awesome Google Hangout full of people. Coincidentally, on the same day as a New York City-based [meetup for people getting familiarized with Hoodie](http://hood.ie/blog/meet-the-hoodies-offline.html).
+
+Soon after, the team created [a Slack channel](http://hood.ie/chat/) and [GitHub repository](https://github.com/hoodiehq/editorial/). GitHub is a social coding site that offers a free storage and collaboration service to customers, which can be conveniently used to store documents, hold discussions, and draft copy, among other things. Think Google Docs, for programmers.
+
+Things got off to an awesome start! Folks were having interesting and sometimes intense discussions. People began writing processes and volunteering for writing and coordinating jobs.
+
+And then, well…and then the holidays came. The funny thing about Open Source is that it’s voluntary, so obviously real life comes first. But the timing of the editorial team getting kicked off right before a series of holidays—Thanksgiving in the US, Christmas, and then New Year, meant that real life held __a lot__ more distractions than quite possibly any other time of the year.
+
+In February, when things had calmed back down, I started poking around to get the editorial team up and going again. A couple weeks later, we have some solid contributions under our belt, and I think we’re making great headway for new people to be able to jump in and get started.
```

Comment:
as mentioned on the Google doc, the sudden shift to I-narrator feels a bit abrupt :)